docs/v1/P4Runtime-Spec.mdk

```diff
@@ -624,21 +624,35 @@ reasons:
    one or more standby slave controllers, which take over controlling the
    devices in case the master controller goes offline.
 
-To support multiple controllers, P4Runtime uses the same streaming channel
-(available via `StreamChannel` RPC) for session management. The workflow is
-described as follows:
-
-* Each controller is assigned a `role_id` and an `election_id`. The `role_id`
-  defines the role (or realm) that the controller is part of. The `election_id`
-  is unique per role and identifies the master for a specific role. At any point
-  in time, the controller with the largest `election_id` for each role is the
-  master and the rest are slaves. Note that the device does not assign a
-  `role_id` and `election_id` to any controller. It is up to an arbitration
-  mechanism outside of the device to decide on the controller roles and the
-  master and slave controllers for each role. The P4Runtime server running on
-  the device only keeps track of the `role_id` and `election_id` of the
-  controllers to determine which connected controller is master at any point of
-  time.
+To support multiple controllers, P4Runtime uses the streaming channel (available
+via `StreamChannel` RPC) for session management. The workflow is described as
+follows:
+
+* Each controller instance (e.g. a controller process) can participate in one or
+  more roles. For each (`device_id`, `role_id`), the controller receives an
+  `election_id`, which may be the same numeric value across multiple roles. For
```

Comment:
could we replace "which may be the same numeric value across multiple roles" with a proposal made by Stefan in the Github issue:

> this election_id can be the same for different roles and/or devices, as long as the tuple (role_id, device_id, election_id) is unique

Based on feedback from Waqar, it seems that the most common case is "same election id advertised to all devices in a given role" and I don't think we should emphasize the other - less frequent - use case ("same election id across roles").

Reply:
Done.